word.js

```diff
@@ -1,40 +1,62 @@
+// require the Letter module
 const Letter = require('./letter');
 
-let Word = (word) => {
+function Word(word) {
   this.word = word;
+  this.displayWord = '';
   this.allLetters = [];
-  this.guessesLetters = [];
   this.attempts = 5;
 
-
-  let blankSpace = () => {
-    allLetters = word.split('');
-    guessesLetters = wordArr.map(function(i) {
-      let objLetter = new Letter(i);
-      allLetters.push(objLetter)
-      return objLetter.blankSpace();
+// generate blank space
+  this.blankSpace = () => {
+    let wordArr = word.split('');
+    // store 'this' in a variable to avoid conflicts
+    let thisWord = this;
+    wordArr.forEach(function(element, index) {
+      let objLetter = new Letter(element);
+      thisWord.allLetters.push(objLetter)
+      thisWord.displayWord += objLetter.getResult();
     });
```

Comment:
this can be written using map, like so:
```javascript
this.allLetters = wordArr.map(function(element, index){
  var currentLetter = new Letter(element)
  thisWord.displayWord += currentLetter.getResult()
  return currentLetter
})```
this is clearer and requirer's less looking around for variable declaration and assignment.